src/mesh/wifi/WiFiAPClient.cpp

```diff
@@ -109,66 +174,17 @@ void deinitWifi()
     LOG_INFO("WiFi deinit\n");
 
     if (isWifiAvailable()) {
+#ifdef ARCH_ESP32
+        WiFi.disconnect(true, false);
+#else
         WiFi.disconnect(true);
-        WiFi.mode(WIFI_MODE_NULL);
+#endif
+        WiFi.mode(WIFI_OFF);
```

Comment:
Same as `WIFI_MODE_NULL`, but that's only defined for ESP32 (https://github.com/espressif/arduino-esp32/blob/master/libraries/WiFi/src/WiFiType.h#L32). Same holds for `WIFI_MODE_STA` vs `WIFI_STA`.